src/vs/workbench/services/issue/browser/issueService.ts

```diff
@@ -26,25 +49,101 @@ export class WebIssueService implements IWorkbenchIssueService {
+			if (!extensionGitHubUrl) {
+				throw new Error(`Unable to find issue reporting url for ${extensionId}`);
 			}
-			throw new Error(`No issue reporting URL configured for ${this.productService.nameLong}.`);
+			const uri = this.getIssueUriFromStaticContent(`${extensionGitHubUrl}/issues/new`, selectedExtension);
+			dom.windowOpenNoOpener(uri);
 		}
 
-		const selectedExtension = this.extensionService.extensions.filter(ext => ext.identifier.value === options.extensionId)[0];
-		const extensionGitHubUrl = this.getExtensionGitHubUrl(selectedExtension);
-		if (!extensionGitHubUrl) {
-			throw new Error(`Unable to find issue reporting url for ${extensionId}`);
+		if (this.productService.reportIssueUrl) {
+			const theme = this.themeService.getColorTheme();
+			const experiments = await this.experimentService.getCurrentExperiments();
+
+			let githubAccessToken = '';
+			try {
+				const githubSessions = await this.authenticationService.getSessions('github');
+				const potentialSessions = githubSessions.filter(session => session.scopes.includes('repo'));
+				githubAccessToken = potentialSessions[0]?.accessToken;
+			} catch (e) {
+				// Ignore
+			}
+
+			// air on the side of caution and have false be the default
+			let isUnsupported = false;
+			try {
+				isUnsupported = !(await this.integrityService.isPure()).isPure;
+			} catch (e) {
+				// Ignore
+			}
+
+			const extensionData: IssueReporterExtensionData[] = [];
+			try {
+				const extensions = await this.extensionManagementService.getInstalled();
+				const enabledExtensions = extensions.filter(extension => this.extensionEnablementService.isEnabled(extension) || (options.extensionId && extension.identifier.id === options.extensionId));
+				extensionData.push(...enabledExtensions.map((extension): IssueReporterExtensionData => {
+					const { manifest } = extension;
+					const manifestKeys = manifest.contributes ? Object.keys(manifest.contributes) : [];
+					const isTheme = !manifest.main && !manifest.browser && manifestKeys.length === 1 && manifestKeys[0] === 'themes';
+					const isBuiltin = extension.type === ExtensionType.System;
+					return {
+						name: manifest.name,
+						publisher: manifest.publisher,
+						version: manifest.version,
+						repositoryUrl: manifest.repository && manifest.repository.url,
+						bugsUrl: manifest.bugs && manifest.bugs.url,
+						displayName: manifest.displayName,
+						id: extension.identifier.id,
+						data: options.data,
+						uri: options.uri,
+						isTheme,
+						isBuiltin,
+						extensionData: 'Extensions data loading',
+					};
+				}));
+			} catch (e) {
+				extensionData.push({
+					name: 'Workbench Issue Service',
+					publisher: 'Unknown',
+					version: '0.0.0',
+					repositoryUrl: undefined,
+					bugsUrl: undefined,
+					extensionData: 'Extensions data loading',
+					displayName: `Extensions not loaded: ${e}`,
+					id: 'workbench.issue',
+					isTheme: false,
+					isBuiltin: true
+				});
+			}
```

Comment:
What type of error is this from? I would perhaps prefer a different type with a different shape to show this info instead of a lot of placeholder data, but this is fine too.

Reply:
updated so `extensionData` (which will actually be seen by the user) will include the information as well. This error happens when extensions don't load from the`extensionEnablementService`